helm/templates/webhook/_webhook.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "connaisseur.namespaceLabels" }}
```

Comment:
heh. I think it's time we refactor our helm charts. This is much better than the current setup.